tests/config/pubsub/tests.yml

```diff
@@ -12,7 +10,7 @@
 componentType: pubsub
 components:
   - component: azure.eventhubs
-    operations: ["publish", "subscribe", "multiplehandlers", "bulkpublish"]
+    allOperations: true
```

Comment:
Please revert these changes. `allOperations` now includes additional operations that these components do not support, such as BulkPublish and BulkSubscribe.

All edits in this file must be reverted, except for your addition of KubeMQ

It seems you actually forgot to add an entry for `kubemq`, so the conformance tests would never actually run against `kubemq`!